src/main/kotlin/net/leanix/vsm/sbomBooster/configuration/RestTemplateConfiguration.kt

```diff
@@ -0,0 +1,15 @@
+package net.leanix.vsm.sbomBooster.configuration
+
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.web.client.RestTemplate
+
+@Configuration
+class RestTemplateConfiguration {
+
+    @Bean
+    fun restTemplate(): RestTemplate {
+
+        return RestTemplate()
```

Comment:
Why rest template? We use Feign in all other projects

Reply:
This was already part of the initial implementation. I just moved it to a bean. We chose this because initially there was only a single request that we needed to perform. Still, I think maybe we need to use the Feign client in the future.